src/gadgets/bits2num.rs

```diff
@@ -218,33 +246,28 @@ mod test {
             let bit2num = Bits2NumChip::<F>::construct(config.bits2num_chip_conf);
 
             let res = match self.mode {
-                BiteMode::BE => { bit2num.convert_be(&mut layouter, assigned_input)?  }
-                BiteMode::LE => { bit2num.convert_le(&mut layouter, assigned_input)? }
+                BiteMode::BE => bit2num.convert_be(&mut layouter, assigned_input)?,
+                BiteMode::LE => bit2num.convert_le(&mut layouter, assigned_input)?,
             };
 
             layouter.constrain_instance(res.cell(), config.pub_input.clone(), 0)?;
 
             Ok(())
         }
-
     }
 
     #[test]
     fn test_bits2num_be_chip() {
-
         let params = 4;
         let input = vec![true, false, true, false];
 
         let circuit = Bits2NumTestCircuit {
             input: input.clone(),
             params,
-            mode: BiteMode::BE
+            mode: BiteMode::BE,
         };
 
-        let answer = input.iter().fold(0, |acc, bit| {
-            acc * 2 + (*bit as u64)
-        });
-        println!("Answer: {}", answer);
+        let answer = 10;
```

Comment:
Think it's better to hard-code the answer in the test, because the code computing it could have the same bug as the gadget! It's also quite easy to do in the head when reading the code in this case.